package/gluon-web-model/Makefile

```
@@ -5,8 +5,8 @@ PKG_NAME:=gluon-web-model
 include ../gluon.mk
 
 define Package/gluon-web-model
-  TITLE:=Minimal Lua web framework derived from LuCI (model support)
-  DEPENDS:=+gluon-web
+	TITLE:=Minimal Lua web framework derived from LuCI (model support)
+	DEPENDS:=+gluon-web
```

I don't think any indentation with tabs instead of spaces would be valid here, though I am not sure why.

I suspect if we want to get rid of spaces in Makefiles, this would be a valid option:
```suggestion
TITLE:=Minimal Lua web framework derived from LuCI (model support)
DEPENDS:=+gluon-web
```